src/views/Editor.vue

```diff
@@ -282,6 +282,12 @@ export default {
 			return null
 		},
 		confirmUnsavedChanges(dialogTitle, noButtonTitle, callback) {
+			// if session is lost, user cannot save
+			if (this.$auth.user === null) {
+				callback(true)
+				return
```

Comment:
Style: I would prefer that if return is not really needed it is not used, this could be simple if else structure instead.

Reply:
I guess I'm used to returning early in Go. Keeps code more flat, but doesn't really matter in this case. Changed.